features/web_bluetooth.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Web Bluetooth API
+category: apps
+bugzilla: 1204396
+firefox_status: under consideration
```

Comment:
I'd say "in-development", since there's a clear indication that Firefox is going to implement it.